src/types/api.js

```diff
@@ -12,12 +12,18 @@
  * @prop {string} [id]
  * @prop {string[]} [references]
  * @prop {string} created
+ * @prop {string} updated
+ * @prop {string[]} tags
+ * @prop {string} text
+ * @prop {string} uri
+ * @prop {string} user
  */
 
 /**
  * TODO - Fill out remaining properties
```

Comment:
is this TODO done?

Reply:
Not yet. Going over the TODOs in this file is still to be done.